source/a_releasenotes.rst

```diff
@@ -2,9 +2,111 @@
 Release notes
 *************
 
-*Release announcement API*
-----------------------------
-Monday March 8th, at 8.00 AM CET we will release an update of our API. The expected downtime is approximately 1 hour. Please note that this means running simulations will be stopped.  
+Release 3Di 8 March 2021
+------------------------
+
+The following has been released:
+
+- Local calculation of water depth maps
+- Extended support for calculations in the Modeller Interface
+- Extended API v3 with boundary conditions & bugfixing
+- Bugfixes in calculation core
+- Update land use map for damage estimations
+
+Download the latest version of the Modeller Interface here. For QGIS users: upgrade the plugin using the plugin panel. The latest version is 1.16. 
```

Comment:
Should `here` be a link?

Reply:
yes, however Richard hasn't built them yet

Reply:
I have tested this, but there is this functionality commented out. so it is a really minor adjustment

Reply:
added the link